src/app/components/popup.ts

```diff
@@ -4,10 +4,13 @@ interface popup {
   headerText: string, 
   popupContentHTML: HTMLElement, 
   closeBtnText: string,
-  isOverlayClickable?: boolean
+  isOverlayClickable?: boolean,
+  additionalBtn?:string,
```

Comment:
You can actually set new optional Object additiinalBtn with properties such as classes value, styles etc